.github/CONTRIBUTING.md

```diff
@@ -1,36 +1,35 @@
 # How to contribute
 
-Third-party patches are essential for keeping every project great. We simply can't access the huge number of use cases. We want to keep it as easy as possible to contribute changes that get things working in your environment. There are a few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things.
+Third-party patches are essential for keeping every project great. We simply can't access the huge number of use cases. We want to keep it as easy as possible to contribute changes that get things working in your environment. There are a few guidelines that we need contributors to follow to have a chance of keeping on top of things.
+
+Our kanban board is public and available [here](https://github.com/orgs/vazco/projects/1).
 
 ## Getting started
 
-- Make sure you have a [GitHub account](https://github.com/signup/free)
+- Make sure you have a [GitHub account](https://github.com/signup/free).
 - Submit a ticket for your issue, assuming one does not already exist.
-  - Clearly describe the issue including steps to reproduce if it is a bug.
-  - Make sure you fill in the earliest version that you know has the issue.
+  - It is OK to comment an already closed issue or review already merged PR.
+    - If needed, we'll reopen it or open a new one.
+  - Clearly describe the issue, including steps to reproduce if it is a bug.
+    - If possible, include a reproduction using our [playground](https://uniforms.tools/playground) or [CodeSandbox template](https://codesandbox.io/s/github/vazco/uniforms/tree/master/reproductions).
+  - Affected versions of the core, bridge, and theme packages you are aware of.
```

Comment:
I think it would sound better with a verb here.

```suggestion
  - Add a list of affected versions of the core, bridge, and theme packages you are aware of.
```

Reply:
Or maybe nest it as a child of the `Clearly describe the issue, including steps to reproduce if it is a bug.` point.